src/Packagist/Api/Client.php

```diff
@@ -165,9 +165,7 @@ protected function request($url)
 
         return $this->httpClient
             ->get($url)
-            ->send()
-            ->getBody(true)
-        ;
```

Comment:
As per the [upgrade documentation](https://github.com/guzzle/guzzle/blob/master/UPGRADING.md#client):

```php
// 3.0
$request = $client->get('/');
$response = $request->send();

// 4.0
$response = $client->get('/');
```